### DISCUSSION TITLE
How to disable login and make verified email needed to login

### DISCUSSION THREAD

**Discussion by HexiDev** [pocketbase/pocketbase#2643] [Q&A]
So I tried doing rules to admin only and the still have access to it so I am not sure if rules will work so I want to know how I can implement that kind of feature and suppend their account

**Comment by ganigeorgiev** [comment]
This is similar to https://github.com/pocketbase/pocketbase/discussions/2630.

There is no API rule for that. At the moment, the only way to disable the login for unverified users completely would be to use PocketBase as framework and attach for example to the [`OnRecordAuthRequest` hook](https://pocketbase.io/docs/event-hooks/#record-auth-api-hooks) (_with the next release there will be support for JS app hooks too_).

**Reply by HexiDev** [reply]
Okey and when will that be (Because I wanna drop this in production 🙃 )

**Reply by ganigeorgiev** [reply]
Sorry, there is no fixed ETA yet and v0.17.0 release may take a while.
